server.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+const app = express();
+const routes = require('./Routes');
+const server = require('http').Server(app);
+var bodyParser = require('body-parser')
+var cookieParser = require('cookie-parser');
+
+const port = 5000;
+
```

Comment:
I am approving it but I think some changes will be required in the future(allow headers or something like that)